### DISCUSSION TITLE
关于预售资金监管 我把地方政府告上了法庭

### DISCUSSION THREAD

**Discussion by hqvv** [WeNeedHome/SummaryOfLoanSuspension#400] [Show and tell]
编辑掉

**Comment by xiejun946** [comment]
我觉得告政府没什么用，直接住进售楼部，样板间，然后停止还贷款就好了。

**Comment by hoicking** [comment]
胆敢状告本官？

**Comment by JACWATA** [comment]
兄die你这样暴露了吧

**Reply by hqvv** [reply]
怕啥  咱又没有说祖国坏话， 再说这些都是祖国给的法定的救济途径，咱也知法守法用法的好市民嘛。 
而且上github 又不需要翻墙，咱也不知道这是外网啊

**Reply by eren-amundsen** [reply]
> 怕啥 咱又没有说祖国坏话， 再说这些都是祖国给的法定的救济途径，咱也知法守法用法的好市民嘛。 而且上github 又不需要翻墙，咱也不知道这是外网啊

支持你

**Reply by xiaer93** [reply]
支持你。加油。

**Reply by yalo-ddaa** [reply]
告上法院的那一刻就已经暴露了，所以没必要再担心。
就跟那些已经去参与维权，却害怕的一样。
